docs/usage/tutorials.md

```diff
@@ -617,3 +617,40 @@ You can then add the `<YOUR_DB_NAME>/` path to your nf-core/taxprofiler database
 </details>
 
 More information on custom KMCP database construction can be found [here](https://bioinf.shenwei.me/kmcp/database/#building-custom-databases).
+
+#### sylph custom database
+
+To build a sylph database, it requires only fasta files.
+
+```bash
+sylph sketch genomes/*.fa.gz
+```
+
+If all genomes are in the same file, you can run:
+
+```bash
+sylph sketch all_genomes.fa
+```
+
+You can then add the `<YOUR_DB_NAME>/` path to your nf-core/taxprofiler database input sheet.
+
+<details markdown="1">
+<summary>Expected files in database directory</summary>
+
+- `sylph`
+  - `.syldb`
```

Comment:
Is that a hidden file, or is it missing a database name at the beginning?

Reply:
This is either `database.syldb` with is the default output when running `sylph sketch` or `<custom_name>.syldb`